R/gh.R

```diff
@@ -177,7 +177,20 @@ gh <- function(endpoint, ..., per_page = NULL, .token = NULL, .destfile = NULL,
   while (!is.null(.limit) && length(res) < .limit && gh_has_next(res)) {
     if (.progress) update_progress_bar(prbr, res)
     res2 <- gh_next(res)
-    res3 <- c(res, res2)
+
+    if (!is.null(names(res2)) && identical(names(res), names(res2))) {
+      res3 <- mapply(           # Handle named array case
```

Comment:
Do we need `SIMPLIFY = FALSE` for `mapply()`?

Reply:
That seems like a good idea